server/JsDbg.WinDbg/Program.cs

```diff
@@ -32,6 +32,7 @@ public class Program {
             string remoteString;
             if (args.Length < 1 || args[0] == "/ask") {
                 // A debugger string wasn't specified.  Prompt for a debug string instead.
+                Console.Write("Hint: You can use '.server npipe:pipe=jsdbg' in WinDbg.")
```

Comment:
might be better to say "You can run the following command to create a remote debugging server in WinDbg: '.server npipe:pipe=jsdbg'"

Reply:
Done